pkg/capabilities/mercury/mercury.go

```diff
@@ -54,7 +54,7 @@ type ReportInfo struct {
 
 // TODO: fix this by adding support for uint64 in value.go
 type FeedReport struct {
-	FeedID               int64  `json:"feedId"`
+	FeedID               string `json:"feedId"`
```

Comment:
What's the motivation behind this change by the way? I ask mainly because I have a task open to convert this to bytes[] which maybe we won't need to do if we use strings here instead?

https://smartcontract-it.atlassian.net/jira/software/c/projects/KS/boards/182?selectedIssue=KS-99

Reply:
String is used to represent a hex-encoded byte array.  Internally it might be better to use []byte or [32]byte, I'm not sure...

Reply:
If we decide to use a string here then we would need to add some validations (starting with "0x", correct number of bytes, all lowercase)

Reply:
Now using [32]byte wherever possible :)